.vscode/settings.json

```diff
@@ -4,7 +4,7 @@
         "typescript"
     ],
     "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true,
+        "source.fixAll.eslint": "explicit"
```

Comment:
why u changed this?

Reply:
VS code made this change automatically. it is the same as true
https://code.visualstudio.com/updates/v1_85#_code-actions-on-save-and-auto